docs/production-deployment/cloud/nexus/operations.mdx

```diff
@@ -0,0 +1,207 @@
+- Handler Namespace
+  - PollNexusTaskQueue
+  - RespondNexusTaskCompleted
+  - RespondNexusTaskFailed
+
+See [Cloud Metrics](/production-deployment/cloud/metrics/reference) for additional details.
+
+## Audit Logging
+
+The following Nexus control plane actions are sent to the Audit Logging integration:
+
+- Create Nexus Endpoint: `CreateNexusEndpoint`
+- Update Nexus Endpoint: `UpdateNexusEndpoint`
+- Delete Nexus Endpoint: `DeleteNexusEndpoint`
+
+See [Audit Logging](/cloud/audit-logging) for additional details.
+
+## Rate Limiting
+
+Nexus requests (commands, polling) are counted as part of the overall Namespace RPS limit in both the caller and handler Namespaces.
+Default Namespace RPS limits are set at 1600 and automatically adjust based on recent usage (over prior 7 days).
+
+## SLOs & SLAs
+
+Nexus requests (commands, polling) have the same latency SLOs and error rate SLAs as other Worker requests in both the caller and handler Namespaces.
+
+See [Availability](/cloud/service-availability) and [SLA](/cloud/sla) for additional details.
+
+## Limits
+
+### Max Nexus Endpoints
+
+By default, each account is provisioned with a max of 10 Nexus Endpoints. 
+You can request further increases beyond the initial 10 Endpoint limit by opening a support ticket.
+
+### Workflow Max Nexus Operations
+
+A single Workflow Execution can have a maximum of 30 in-flight Nexus Operations and 30 total Nexus Operations (as Public Preview does not yet remove completed Nexus Operations from mutable state).
+After that limit is reached, no more Nexus Operations will be processed for that Workflow Execution.
+
+### Nexus Request Handler Timeout
+
+Nexus Operation handlers have less than 10 seconds to process a single Nexus start or cancel request.
+Handlers should observe the context deadline and ensure they do not exceed it.
+This includes fully processing a synchronous Nexus operation and starting an asynchronous Nexus operation, for example one that starts a Workflow.
+If a handler doesn’t respond within a context deadline, a context deadline exceeded error will be tracked in the caller Workflow’s pending Nexus operations, and the Nexus Machinery will retry the Nexus request with an exponential backoff policy.
+
+### Nexus Operation Maximum Duration
+
+Each Nexus Operation has a maximum ScheduleToClose duration of 60 days, which is most applicable to asynchronous Nexus Operations that are completed with an asynchronous callback using a separate Nexus request from the handler back to the caller Namespace. 
+The 60 day maximum is forward-compatible with completion callbacks that may be signed with a single use token such that asymmetric encryption keys may be rotated in a timely fashion. 
+While the caller of a Nexus Operation can configure the ScheduleToClose duration to be shorter than 60 days, the maximum duration can not be extended beyond 60 days and will be capped by the server to 60 days.
```

Comment:
interesting, so if someone requests 90 days, we don't fail the request, but instead truncate the maximum to 60 days? cc @bergundy

Reply:
talked offline with Phil/Roey, this is accurate, but we should probably change the behavior to have it fail sooner rather than later since it may surprise someone who passes in a higher value and doesn't realize it is getting truncated